assets/i18n/qqq.json

```diff
@@ -135,7 +135,7 @@
 	"I18N_FOOTER_FORUM": "Text shown in the footer. - Link to the forum page.\n{{Identical|Forum}}",
 	"I18N_FOOTER_GET_INVOLVED": "Text shown in the footer. - Link to the contact page that explains how to contribute to the Oppia foundation.\n{{Identical|Get involved}}",
 	"I18N_FOOTER_GET_STARTED": "Text shown in the footer. - Link to the get started page. See I18N_GET_STARTED_PAGE_TITLE\n{{Identical|Get started}}",
-	"I18N_FOOTER_OPPIA_FOUNDATION": "Text shown in the footer. - Link to the Oppia foundation tab in the about page. See I18N_ABOUT_PAGE_TITLE\n\nDon't translate this message !",
+	"I18N_FOOTER_OPPIA_FOUNDATION": "Text shown in the footer. - Link to the Oppia Foundation website. See I18N_ABOUT_PAGE_TITLE\n\nDon't translate this message!",
```

Comment:
Do you know why the "Don't translate this messsage!" bit is there? It seems to me like if we don't want people to translate a string, it shouldn't be in the translation files in the first place...?

Reply:
Hmm... Looks like you introduced it two months ago in #3685? 

https://github.com/oppia/oppia/blame/develop/assets/i18n/qqq.json#L138

Though, that's a squashed merge commit.

It looks like it was actually put in by TranslateWiki here: https://github.com/oppia/oppia/commit/b90acf486178be57fadb8e16fcb96f59df918682

Not really sure!

Reply:
Do you think we should just remove that? Seems quite odd.

Reply:
Oh, perhaps it means that that string should always derive from `I18N_ABOUT_PAGE_TITLE` for parity?